test/image.test.js

```diff
@@ -53,18 +84,48 @@ describe('Image', function () {
     };
 
     try {
-      img.src = png + 's';
+      img.src = png_clock + 's';
     } catch (err) {
       assert.fail('error did not invoke onerror(): ' + err);
     }
 
     assert.strictEqual(1, onerrorCalled);
-    assert.strictEqual(img.src, png + 's');
+    assert.strictEqual(img.src, png_clock + 's');
     assert.strictEqual(false, img.complete);
 
     assert.ok(error instanceof Error, 'did not invoke onerror() with error');
   });
 
+  it('test Image#onerror multiple calls', function() {
+    var img = new Image
+      , onloadCalled = 0;
+
+
+    img.onload = function() {
+      assert.fail('called onload');
+    };
+
+    img.onerror = function() {
+      ++onloadCalled;
```

Comment:
Mind changing to `onloadCalled += 1` for consistency with the rest of the file? I personally think it feels a bit weird to put them in front of the variable instead of after, although I understand that technically you don't want it to retain the old value and return that from this particular expression... There is some debate on wether to use `++` and `--` at all, I think that Crockford recommends against it in "Javascript: the good parts". I think that in this case `onloadCalled += 1` shows the intent very clearly.